src/lib/molecules/NavigationMenu/index.js

```diff
@@ -36,11 +37,14 @@ function NavigationMenu({ data, value, onChange, disabled }) {
             children={
                 <button className="bc-navigation-menu" disabled={disabled}>
                     <div className="bc-navigation-menu_head">
-                        {title.map((item, index) => (
-                            <span key={index} className="bc-navigation-menu_layer">
-                                {item}
-                            </span>
-                        ))}
+                        {title.map((item, index, arr) => (
```

Comment:
Why do you take here `arr` argument? It's not used in the scope of the `map` function.